app/models/miq_schedule_worker/jobs.rb

```diff
@@ -64,6 +64,12 @@ def infra_conversion_dispatcher_dispatch
     end
   end
 
+  def image_import_pvc_to_pvs_dispatcher_dispatch
+    if ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ImageImportWorkflow::ImportDispatcher.waiting?
+      queue_work_on_each_zone(:class_name => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ImageImportWorkflow::ImportDispatcher", :method_name => "dispatch", :task_id => "job_dispatcher", :priority => MiqQueue::HIGH_PRIORITY)
```

Comment:
> `queue_work_on_each_zone`

@iv1111 I thought you wanted one of these for the whole region, not per zone (that would be just `queue_work()`)

Reply:
you're right, I will change now